WooCommerce/Classes/ViewRelated/Editor/AztecUIConfigurator.swift

```diff
@@ -77,6 +78,9 @@ private extension AztecUIConfigurator {
         textView.textAttachmentDelegate = textViewAttachmentDelegate
         textView.backgroundColor = .basicBackground
         textView.textColor = .systemColor(.label)
+        textView.blockquoteBackgroundColor = .neutral(.shade5)
+        textView.blockquoteBorderColor = .listIcon
```

Comment:
What does the semantic name "listIcon" mean? I've seen this being used in labels too. It's odd because it seems intended to be used for [tables and lists only](https://woomobiledesignhandbook.wordpress.com/colors/).

Reply:
I borrowed the styles from WPiOS: https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift#L161-L163

I'm not super sure why though. if @Garance91540 has another semantic color that works for the blockquote border, please lemme know!

Reply:
please ignore the last comment @Garance91540! I updated to use the Woo purple in https://github.com/woocommerce/woocommerce-ios/pull/1986/commits/7f5cd15d3fd2e12414c6be2ddac4aa182502a8a7